src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml

```diff
@@ -181,7 +181,7 @@
                     Height="14"
                     HorizontalAlignment="Center"
                     VerticalAlignment="Center"
-                    Style="{StaticResource QuestionIcon}"
+                    Style="{StaticResource InformationIcon}"
```

Comment:
Your screen shot shows a question mark still?

Reply:
> Your screen shot shows a question mark icon still?

it was a naming glitch that I missed, now the NodeAutocomplete uses the InformationIcon and the Preferences uses the QuestionIcon

Reply:
Gotcha